COVAXIN (#924)
greenelab/covid19-review: content/22.vaccines.md

Thus far, over 63 million people have been vaccinated against COVID-19 in India [@url:https://www.mygov.in/covid-19/].
A broad range of COVID-19 vaccine candidates are under investigation in order to respond to the COVID-19 pandemic.
In India, the Covaxin vaccine produced by Bharat Biotech received emergency authorization on January 3rd, 2021, despite the lack of phase III data until March 3rd [@url:india-covid-19-vaccine.html; @url:https://www.bharatbiotech.com/images/press/covaxin-phase3-efficacy-results.pdf].
Following the release of the phase III data indicating 81% efficacy, Zimbabwe authorized the use of Covaxin [@url:http://www.xinhuanet.com/english/2021-03/04/c_139783893.htm].
In addition to COVAXIN®, Bharat Biotech has also developed an adenovirus vector-based intranasal BBV154 vaccine candidate that induces neutralizing IgG, mucosal IgA, and T cell responses [@url:https://www.bharatbiotech.com/intranasal-vaccine.html]. 
In February, 2021, Bharat Biotech received approval from Indian officials to commence a phase I study of this intranasal chimpanzee-adenovirus (ChAd) vectored SARS-CoV-2-S vaccine [@url:https://www.bharatbiotech.com/intranasal-vaccine.html]. 

> Another Indian pharmaceutical industry, Zydus Cadila is developing India’s first indiginous DNA vaccine candidate ZyCoV-D [@doi:10.1101/2021.01.26.428240]  which has completed phase I and II clinical trials and has received an approval from Drugs Controller General of India (DCGI) to conduct a phase III clinical trial [@url:https://vaccine.icmr.org.in/covid-19-vaccine].

Reviewer: ```suggestion
Another Indian pharmaceutical industry, Zydus Cadila is developing India’s first indigenous DNA vaccine candidate ZyCoV-D [@doi:10.1101/2021.01.26.428240]  which has completed phase I and II clinical trials and has received approval from Drugs Controller General of India (DCGI) to conduct a phase III clinical trial [@url:https://vaccine.icmr.org.in/covid-19-vaccine].
```

<!--- https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7561304/ BBIBP-CorV --->
<!--- https://pubmed.ncbi.nlm.nih.gov/32778225/ --->

> ###COVAXIN/BBV152

Reviewer 2: ```suggestion
### COVAXIN/BBV152
```

India, the biggest producer of vaccines globally, has developed COVAXIN® an indigenous COVID-19 vaccine researched and manufactured by Bharat Biotech International Ltd. in collaboration with the Indian Council of Medical Research (ICMR) - National Institute of Virology (NIV). 

> Bharat Biotech reported 80.6% vaccine efficacy for its whole virion inactivated  BBV152 (COVAXIN®) vaccine candidate in phase 3 clinical trials [@url:https://www.bharatbiotech.com/images/press/covaxin-phase3-efficacy-results.pdf, https://www.biogenetech.co.th/wp-content/uploads/2021/03/5-Ocugen.pdf]. 

Reviewer 3: Haha go for it :D

> It was reported in _The Lancet_ that the BBV152 vaccine candidate adjuvanted with alum and a Toll-like receptor 7/8 (TLR7/8) agonist is safe, immunogenic, and induces Th1-skewed memory T-cell responses upon immunization [@doi:gh7597].

Author: Thanks! Ronan helped me with that

> Importantly, sera from individuals vaccinated with COVAXIN^®^ efficiently neutralized the B.1.1.7 variant and the heterologous strain of SAR-CoV-2 [@doi:10.1101/2021.01.26.426986].

Reviewer 3: That was something I was wondering too, I assumed that meant the original or wild type strain, but I wasn't sure.

> Importantly, sera from individuals vaccinated with COVAXIN^®^ efficiently neutralized the B.1.1.7 variant and the heterologous strain of SAR-CoV-2 [@doi:10.1101/2021.01.26.426986].

Author: This is a different strain hCoV-19/India/2020Q111 (unclassified cluster)

Russia is developing other vaccine candidates and has approved a third vaccine, CoviVac, which is an inactivated vaccine produced by the Chumakov Centre in Moscow, despite the fact the clinical trials have yet to begin [@url:https://www.reuters.com/article/us-health-coronavirus-russia-vaccine-idUSKBN2AK07H].

In Asia, China and India are the main COVID-19 vaccination developers and providers.

> Thus far, over 63 million people have been vaccinated against COVID-19 vaccine in India [@url:https://www.mygov.in/covid-19/].

Reviewer: ```suggestion
Thus far, over 63 million people have been vaccinated against COVID-19 in India [@url:https://www.mygov.in/covid-19/].
```

A broad range of COVID-19 vaccine candidates are under investigation in order to respond to the COVID-19 pandemic.

> In India, the Covaxin vaccine produced by Bharat Biotech received emergency authorization on January 3rd, 2021, despite the lack of phase III data until March 3rd [@url:india-covid-19-vaccine.html; @url:https://www.bharatbiotech.com/images/press/covaxin-phase3-efficacy-results.pdf].

Reviewer 3: Is it usually written as just Covaxin, or in all caps? Just wondering for consistency :)

Author: Wherever I have seen it, it is in all CAPS

Following the release of the phase III data indicating 81% efficacy, Zimbabwe authorized the use of Covaxin [@url:http://www.xinhuanet.com/english/2021-03/04/c_139783893.htm].

> In addition to COVAXIN®, Bharat Biotech has also developed an adenovirus vector-based intranasal BB154 vaccine candidate that induces neutralizing IgG, mucosal IgA, and T cell responses [@url:https://www.bharatbiotech.com/intranasal-vaccine.html]. 

Reviewer: Is BB154 missing a V or is this a different vaccine than the line below but with a very close name?

Author: Sorry, missed the V in BBV154

Reviewer 3: ```suggestion
In addition to COVAXIN®, Bharat Biotech has also developed an adenovirus vector-based intranasal BBV154 vaccine candidate that induces neutralizing IgG, mucosal IgA, and T cell responses [@url:https://www.bharatbiotech.com/intranasal-vaccine.html]. 
```

Reviewer 3: Added!

Reviewer: Yay! I was hoping that was the case because my eyes aren't shart enough to handle two names that close 😆

> In February, 2021, Bharat Biotech received approval from Indian officials to commence a phase I study of an intranasal chimpanzee-adenovirus (ChAd) vectored SARS-CoV-2-S vaccine called BBV154 [@url:https://www.bharatbiotech.com/intranasal-vaccine.html]. 

Reviewer: If this is the same vaccine as the line above, maybe just rephrase slightly to make that more clear? like "their" instead of "an" before "intranasal chimpanzee-adenovirus vectored"

Author: I am new to GitHub. Struggling to get back to the original content and edit this or give my information as you requeste. Sorry about this. Thanks for your patience.

Reviewer: I absolutely understand @NAIKA86, it's quick the learning curve! You can use [these directions](https://github.com/greenelab/covid19-review/blob/master/CONTRIBUTING.md#the-how-to-of-review) if you want, they might make it a little easier!

Reviewer: For the author info, you can just put it in a comment and I can put it into the file, if that's easier!

Author: github: NAIKA86
name: Amruta Naik
initials: AN
orcid:https://orcid.org/0000-0003-0673-2643
twitter: (optional)
email: NAIKA@chop.edu
Do you agree to the code of conduct? Yes
affiliation(s): Children's Hospital of Philadelphia
Do you have any conflicts of interest? No
Do you have any funding you'd like listed? (And what is it?) No

Reviewer: Awesome! It's updated!

Reviewer: ```suggestion
In February, 2021, Bharat Biotech received approval from Indian officials to commence a phase I study of this intranasal chimpanzee-adenovirus (ChAd) vectored SARS-CoV-2-S vaccine [@url:https://www.bharatbiotech.com/intranasal-vaccine.html]. 
```

> Another Indian pharmaceutical industry, Zydus Cadila is developing India’s first indiginous DNA vaccine candidate ZyCoV-D [@doi:10.1101/2021.01.26.428240]  which has completed phase I and II clinical trials and has received an approval from Drugs Controller General of India (DCGI) to conduct a Phase III clinical trial [@url:https://vaccine.icmr.org.in/covid-19-vaccine].

Reviewer 3: ```suggestion
Another Indian pharmaceutical industry, Zydus Cadila is developing India’s first indiginous DNA vaccine candidate ZyCoV-D [@doi:10.1101/2021.01.26.428240]  which has completed phase I and II clinical trials and has received an approval from Drugs Controller General of India (DCGI) to conduct a phase III clinical trial [@url:https://vaccine.icmr.org.in/covid-19-vaccine].
```

Reviewer: ```suggestion
Another Indian pharmaceutical industry, Zydus Cadila, is developing India’s first indigenous DNA vaccine candidate, ZyCoV-D [@doi:10.1101/2021.01.26.428240], which has completed phase I and II clinical trials and has received an approval from Drugs Controller General of India (DCGI) to conduct a phase III clinical trial [@url:https://vaccine.icmr.org.in/covid-19-vaccine].
```

> Moreover, Indian-based Biological E.Limited has partnered with U.S. based biopharmaceutical company Dynavax Technologies Corporation and Baylor College of Medicine, Texas, to initiate a phase I/II clinical trial in India for its COVID-19 subunit vaccine (consists of the RBD of the spike protein of SARS-CoV-2) [@url:https://vaccine.icmr.org.in/covid-19-vaccine]. 

Reviewer: ```suggestion
Moreover, Indian-based Biological E.Limited has partnered with U.S.-based biopharmaceutical company Dynavax Technologies Corporation and Baylor College of Medicine, Texas, to initiate a phase I/II clinical trial in India for its COVID-19 subunit vaccine, which consists of the RBD of the spike protein of SARS-CoV-2 [@url:https://vaccine.icmr.org.in/covid-19-vaccine]. 
```
